custom-resources/cert-manager-issuer.tf

```diff
@@ -28,7 +32,7 @@ resource "kubernetes_manifest" "cluster_issuer" {
         "privateKeySecretRef" = {
           "name" : "letsencrypt-prod-issuer-account-key"
         },
-        "solvers" = concat([for zone in var.core_services.azure_dns_zones.dns_zones : {
+        "solvers" = concat([for zone in var.core_services.azure_dns_zones.dns_zone : {
```

Comment:
we should double-check that the `solvers` expect the whole zone id instead of just the zone name.

Reply:
Yea it expects the string. Currently testing a fix for this.